pylint/checkers/base/comparison_checker.py

```diff
@@ -58,6 +58,13 @@ class ComparisonChecker(_BasicChecker):
             "comparison-with-itself",
             "Used when something is compared against itself.",
         ),
+        "R0125": (
+            "Comparison between constants: '%s %s %s' is also a constant, use it directly",
```

Comment:
```suggestion
            "Comparison between constants: '%s %s %s' is always %s remove the comparison and consider a refactor",
```
Wondering what a better message could be here. (I know I'm the one who suggested this one 😄)

``comparison-of-constants:14:9:14:21:foo:"Comparison between constants: 'None == None' is always true  remove the comparison and consider a refactor":HIGH``

Thoughts ?

Reply:
How would you suggest we (safely) evaluate the expression?

Reply:
We could use inference for that (but keep the message confidence HIGH as it's only the content of the message that is affected)

Reply:
I'm not sure it's worth it (yet). We don't yet know what kind of code will be met by this lint. 

Better let end-users tell us if/when they think our message to them is falling short. We'll know what to do when we know why we're doing it.

Reply:
```suggestion
            "Comparison between constants: '%s %s %s' is also a constant, remove the comparison and consider a refactor",
```